content/waku/2024-gantt.md

```diff
@@ -24,92 +24,125 @@ gantt
-        TBC E2e reliability protocol Status integration: crit, e2e-rel-g, after e2e-rel-r, 6w
-    section golang eng 3
-        Store v3 (go-waku client only): crit, storev3-g, 2024-02-26, 2024-05-24
-        %% Estimate TBC - assuming parallel work possible
-        TBC Review connection management: crit, review-conn-mgmt-2, after storev3-g, 8w
-        Sharding peer mgmt and discovery hardening (go-waku): crit, sh-peer-mgmt-g, after review-conn-mgmt-2, 8w
-    section golang eng 4
-        Store v3 (sync): crit, 2024-02-08, 2024-04-26
-        Reliability Protocol for Resource-Restricted Clients (go): crit, rel-reqres-g, 2024-05-13, 10w
-        (metric) Count store messages: count-store-msg, after rel-reqres-g, 2w
-        Investigation and fixing of bugs discovered during dogfooding/usage/simulations: go-bugs-2, after count-store-msg, 8w
-    section golang eng 5
-        DoS protection for req-res protocols (go-waku client only): crit, dosreqresg, 2024-05-20, 4w
-        (telemetry) Multicast message reliability: telem-m-msg-rel, after dosreqresg, 4w
-    section golang eng 6
-        (telemetry) direct message reliability: telem-d-msg-rel, 2024-05-13, 6w
-        (telemetry) Measure Bandwidth: telem-bandwidth, after telem-d-msg-rel, 8w
-    section test eng 1
-        Peer and connection management tests: sim-conn-mgmt, 2024-05-13, 4w
-        (simulation) Functionality and stress test store v3: sim-storev3, after sim-conn-mgmt, 4w
-        %% This is Store Service Upgrade - item (2) in DST simulation - start with small scale to get faster results
-        (simulation) Compare store topologies: sim-store-cmp, after sim-storev3, 6w
-        (simulation) relay reliability performance impact: sim-relay-rel, after sim-store-cmp sim-req-res rel-relay-g rel-relay-n, 4w
-        (simulation) req-res reliability performance impact: sim-reqres-rel, after sim-relay-rel rel-reqres-g, 6w
-    section research eng 1
-        End-to-end reliability protocol - PoC: crit, e2e-rel-r, 2024-05-23, 20w
-    section research eng 2
+    section Chat.plopezlpz
+        (testing) Custom shard impl of Communities: test-custom-shard, 2024-08-26, 2024-09-30
+    section Chat.kaichaosun
+        Review MVDS usage and fail path: crit, mvds, 2024-08-01, 2024-09-30
+        %% TODO estimate
+        Minimal Community Specification and Implementation: min-com-spec, after mvds, 12w
+    section Go.richard-ramos
+        Implement RLN smart contract for paid, multilevel memberships: rln-sc-d, 2024-09-01, 6w
+        %% TODO: Estimate
+        Nwaku in Status Desktop: nwaku-status-desktop, after rln-sc-d, 12w
+    section Go.chaitanyaprem
+        DoS Protection for Req-Res Protocols: crit, dosreqres-g, 2024-08-01, 2024-09-15
+        Sharding peer mgmt and discovery hardening: crit, sh-peer-mgmt-g, 2024-08-26, 2024-09-12
+        %% TODO review estimate
+        Review usage of content topics in Status Chat and Communities protocol: cont-topic-r, after sh-peer-mgmt-g, 12w
```

Comment:
In our recent discussion we settled on `10w`. Since there are significant risks and unknowns, perhaps leaving this at `12w` is wise.